cfgov/jinja2/v1/_includes/organisms/sidebar-breakout.html

```diff
@@ -0,0 +1,118 @@
+{# ==========================================================================
+
+   sidebar_breakout.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates Sidebar Breakout markup when given:
+
+   settings:               An object used to customize the markup.
+
+   settings.heading:       A string containing heading text.
+
+   settings.image:         An object used to customize the image
+                           markup.
+
+   settings.intro.body:    A string containing body text.
+
+   settings.intro.heading: A string containing heading text.
+
+   settings.related_posts: An object used to customize the related posts
+                           markup.
+
+   settings.text_intro:    The text intro markup.
+
+   ========================================================================== #}
+
+{% macro render(settings) %}
+<div class="o-sidebar-breakout
+            content__flush-bottom">
+    <div class="content_intro">
+        <div class="content-l">
+            <section class="o-sidebar-breakout_col
+                            content-l_col
+                            content-l_col-2-3">
+                {% if caller %}
+                    {{ caller() }}
+                {% else %}
+                    {{ settings.text_intro }}
+                {% endif %}
+            </section>
+            <aside class="o-sidebar-breakout_col
+                          content-l_col
+                          content-l_col-1-3
+                          block
+                          block__flush-top
+                          block__flush-bottom">
+                {{ _sidebar(settings) }}
+            </aside>
+        </div>
+    </div>
+</div>
+<div class="content_line"></div>
```

Comment:
Is this just a horizontal rule? We have a rule atom now, `a-rule-break`, that goes at the top of the organism/molecule. See https://github.cfpb.gov/flapjack/Modules-V1/wiki/Atoms#grey-rule-line. I forgot about `content_line` though. The rule break atom will need to be reconciled with CF at some point. Also, @jimmynotjim suggested in another PR to make this an `<hr>`